apps/fabric-website-resources/src/AppDefinition.tsx

```diff
@@ -14,19 +14,17 @@ const propertiesTableMargins = mergeStyles({
 });
 
 function loadReferences(): IAppLink[] {
-  const pageList: IReferencesList = require('@uifabric/api-docs/lib/pages/references/list.json');
+  const requireContext = require.context('@uifabric/api-docs/lib/pages/references', false, /\w+\.page\.json$/);
 
-  return pageList.pages.map(pageName => ({
-    component: () => (
-      <ApiReferencesTableSet
-        className={propertiesTableMargins}
-        jsonDocs={require('@uifabric/api-docs/lib/pages/references/' + pageName + '.page.json')}
-      />
-    ),
-    key: pageName,
-    name: pageName,
-    url: '#/examples/references/' + pageName.toLowerCase()
-  }));
+  return requireContext.keys().map(pagePath => {
+    const pageName = pagePath.match(/(\w+)\.page\.json/)![1];
+    return {
+      component: () => <ApiReferencesTableSet className={propertiesTableMargins} jsonDocs={requireContext(pagePath)} />,
+      key: pageName,
+      name: pageName,
+      url: '#/examples/references/' + pageName.toLowerCase()
```

Comment:
Is this going to work in all cases? It may be best to slugify this so we don't get "%20" in urls.

Reply:
Interesting point. @natalieethell Can these page names ever have spaces?

Reply:
Nope they'll never have spaces because they're interface, class, or enum names!